docs/gui/liquids.rst

```diff
@@ -2,41 +2,20 @@ gui/liquids
 ===========
 
 .. dfhack-tool::
-    :summary: Interactively paint liquids or obsidian onto the map.
-    :tags: untested fort armok map
+    :summary: Interactively paint liquids onto the map.
+    :tags: fort armok map
 
-This tool is a gui front-end to `liquids` and works similarly, allowing you to
-add or remove water/magma, and create obsidian walls & floors.
+This tool is a gui for spawning water or magma. Allowing you to click and paint
```

Comment:
Sad to see the removal of obsidian, flows, river sources, etc. - those should be mentioned in `changelog.txt` for sure.

Reply:
To follow up on this, those are planned for a v2 of this script